ocs_ci/ocs/resources/noobaa.py

```diff
@@ -0,0 +1,96 @@
+import base64
+import logging
+import boto3
+
+from ocs_ci.ocs.ocp import OCP
+
+logger = logging.getLogger(name=__file__)
+
+
+class NooBaa(object):
+    """
+    Wrapper class for NooBaa's S3 service
+    """
+
+    _s3_resource = None
+    _ocp_resource = None
+
+    def __init__(self):
+        """
+        Constructor for the NooBaa class
+        """
+        ocp_obj = OCP(kind='noobaa', namespace='noobaa')
+        results = ocp_obj.get()
+        endpoint = (
```

Comment:
do we need this parameter as a class attribute for later usage?
(i.e. use self.endpoint?)

Reply:
Not as far as I can tell. AFAIK, the only use we have for the endpoint is for the initial S3 resource creation.